### PR TITLE
data: theVatsal3802

### PR DESCRIPTION
**Pull request by theVatsal3802**

### CODE REVIEW

File: data/theVatsal3802.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/thevatsal3802",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "GitHub: Will be happy to colab on open-source projects",
+      "url": "https://github.com/theVatsal3802",
+      "icon": "FaGitHub"
+    },
```
Comment (Tejender1521) [inline data/theVatsal3802.json:27]: }

Comment (Tejender1521) [reply data/theVatsal3802.json:27]: Remove , at the end to validate JSON

**Review by Tejender1521 — CHANGES_REQUESTED** [review]
Remove , at line # 27

File: data/theVatsal3802.json (hunk 1 above)
Comment (theVatsal3802) [reply data/theVatsal3802.json]: Done

**Review by iamthecloverly — DISMISSED** [review]

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

File: data/theVatsal3802.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/thevatsal3802",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "GitHub: Will be happy to colab on open-source projects",
+      "url": "https://github.com/theVatsal3802",
+      "icon": "FaGitHub"
```
Comment (kumarsonsoff3) [inline data/theVatsal3802.json:26]:
```suggestion
      "icon": "FaGithub"
```

File: data/theVatsal3802.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/adhiya_vatsal_3",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Let's Connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/vatsal-adhiya-206479202/",
+      "icon": "FaLinkedIn"
```
Comment (kumarsonsoff3) [inline data/theVatsal3802.json:16]:
```suggestion
      "icon": "FaLinkedin"
```

**Review by kumarsonsoff3 — COMMENTED** [review]
I recommend updating the **invalid icons** as suggested, else the default globe icon will be implemented for both the links. 

Rest looks good to me! 👍

File: data/theVatsal3802.json (hunk 3 above)
Comment (theVatsal3802) [reply data/theVatsal3802.json]: Done

File: data/theVatsal3802.json (hunk 2 above)
Comment (theVatsal3802) [reply data/theVatsal3802.json]: Done

**Review by Tejender1521 — APPROVED** [review]
Now everything looks good and ready to be merged

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge, Thanks for your Contribution! 🎉 

<p align="center"> </p>

**Pull request merged**